cms/djangoapps/contentstore/views/transcript_settings.py

```diff
@@ -254,3 +261,32 @@ def transcript_upload_handler(request, course_key_string):
             )
 
     return response
+
+
+@login_required
+@require_http_methods(["DELETE"])
+def transcript_delete_handler(request, course_key_string, edx_video_id, language_code):
+    """
+    View to delete a transcript file.
```

Comment:
Should we mention that this view is specifically for video uploads page?

Reply:
No it can be used in any page.